english/covid-19/0/0-BB.md

```diff
@@ -1,7 +1,7 @@
-Great! Which game will you play?
+Great! Which game 🎲 will you play?
 
-[Rock Paper Scissor](0-BBA.md)
+[🪨📄✂️ Rock Paper Scissor](0-BBA.md)
 
-[Snake & Ladder](../WIP.md)
+[🐍🪜 Snake & Ladder](../WIP.md)
```

Comment:
What is this sign mean?

Reply:
Emojis symbolizing a snake and a ladder as "Snake and Ladder"

Reply:
Does it only have question mark sign, try to use the emoji that could show on github
![image](https://user-images.githubusercontent.com/37890026/118704582-47819300-b7e5-11eb-885e-2f2e7a450b7f.png)